texas-hearing-institute/App.tsx

```diff
@@ -1,21 +1,70 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { Pressable, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Button } from 'react-native-elements';
 import tw from 'tailwind-react-native-classnames';
+import Icon from 'react-native-vector-icons/FontAwesome';
 
+
+// LOOK AT -TTS
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text style={tw`text-3xl font-bold underline`}>Open up App.tsx to start working on your app!</Text>
+    <View style={[styles.container, 
```

Comment:
to keep our code modular, let's create a dedicated `Home.tsx` component and move all of this code into that file (then call it in App())